qiskit/transpiler/preset_passmanagers/level3.py

```diff
@@ -299,12 +295,9 @@ def _unroll_condition(property_set):
         else:
             pre_optimization = common.generate_pre_op_passmanager(remove_reset_in_zero=True)
 
-    if scheduling_method is None or scheduling_method in {"alap", "asap"}:
-        sched = common.generate_scheduling(
-            instruction_durations, scheduling_method, timing_constraints, inst_map, target=target
-        )
-    elif isinstance(scheduling_method, PassManager):
+    if isinstance(scheduling_method, PassManager):
```

Comment:
This is definitely not needed and a `PassManager` can never make it here. But in the interest of one logical change per PR lets remove this in a follow up.